demo/index.html

```diff
@@ -1,80 +1,77 @@
 <!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
 <html>
   <head>
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
 
-    <title>Performance Benchmark Demo</title>
+    <title>Test Benchmark Demo</title>
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="../performance-benchmark.html">
+    <link rel="import" href="../test-benchmark.html">
+    <link rel="import" href="../../paper-styles/demo-pages.html">
+    <style is="custom-style">
+      body {
+        height: 100vh;
+      }
+
+      h3 {
+        text-align: center;
+        color: var(--paper-blue-grey-500);
+      }
+    </style>
   </head>
   <body>
+    <h3>Tap anywhere to begin benchmarking</h3>
 
-    <performance-benchmark runs-per-test="10" auto-start>
-
-      <template data-name="Compute Fibonacci sequence to n = 99999"
-                data-tags="alpha beta">
-        <script>
-          StartTest();
-          var a = 0, b = 1, n = 99999, t;
-          while (n-- > 0) {
-            t = a;
-            a = b;
-            b += t;
-          }
-          EndTest();
-        </script>
-      </template>
-
-      <template data-name="Native button disabled behavior"
-                data-tags="alpha gamma">
-        <button id="NativeButton">Native Button</button>
-        <script>
-          StartTest();
-          NativeButton.disabled = true;
-          NativeButton.removeAttribute('disabled');
-          EndTest();
-        </script>
-      </template>
-
-      <template data-name="Measure document.body.offsetWidth"
-                data-tags="gamma beta">
-        <script>
-          StartTest();
-          document.body.offsetWidth;
-          EndTest();
+    <test-benchmark runs-per-test="100" random-order>
+      <template data-name="paper-button@v1.0.0">
+        <link rel="import" href="//polygit.org/paper-button+polymerelements+v1.0.0/components/paper-button/paper-button.html">
+        <script type="benchmark">
```

Comment:
Could this function live in a js helper? Something like `setup('paper-button')`, and it sets up this listener?